docs/static/monitoring/monitoring-ea-dashboards.asciidoc

```diff
@@ -0,0 +1,109 @@
+Add info to help them decide which one they need.  
+
+**Prerequisites**
+
+Complete these steps as you prepare to collect and ship monitoring data for dashboards:
+
+[[disable-default-db]]
+.Disable default collection of {ls} monitoring metrics
+[%collapsible]
+====
+include::monitoring-prereq-disable-default.asciidoc[]
+====
+
+[[define-cluster__uuid-db]]
+.Specify the target cluster_uuid (optional)
+[%collapsible]
+====
+include::monitoring-prereq-define-cluster.asciidoc[]
+====
+
+
+[discrete]
+[[install-and-configure-db]]
+=== Install and configure {agent}
+
+Install and configure {agent} to collect {ls} monitoring data for dashboards.
+We'll walk you through the process in these steps:
+
+* <<add-logstash-integration-ead>>
+* <<add-agent-to-fleet-ead>>
+* <<view-assets-ead>>
+* <<view-data-dashboard>>
+
+[discrete]
+[[add-logstash-integration-ead]]
+==== Add the {agent} {ls} integration to monitor host logs and metrics
+
+. Go to the {kib} home page, and click **Add integrations**.
++
+--
+[role="screenshot"]
+image::images/kibana-home.png[{kib} home page]
+--
+
+. In the query bar, search for **{ls}** and select the integration to see more
+details.
+
+. Click **Add {ls}**.
+
+. Configure the integration name and add a description (optional).
+
+. Configure the integration to collect logs. 
+* Make sure that **Logs** is turned on if you want to collect logs from your {ls} instance. Be sure that the required settings are correctly configured.
+* Under **Logs**, modify the log paths to match your {ls} environment.
+
+. Configure the integration to collect metrics.
+* Make sure that **Metrics (Technical Preview)** is turned on, and **Metrics (Stack Monitoring)** is turned off.
+* Under **Metrics (Technical Preview)**, make sure the {ls} URL setting
+points to your {ls} instance URLs. +
+By default, the integration collects {ls}
+monitoring metrics from `https://localhost:9600`. If that host and port number are not
+correct, update the `Logstash URL` setting. If you configured {ls} to use encrypted
+communications and/or a username and password, you must access it via HTTPS, and expand the **Advanced Settings** options, and fill in with the appropriate values for your {ls} instance.
+
+. Click **Save and continue**. +
+This step takes a minute or two to complete. When
+it's done, you'll have an agent policy that contains a system integration policy
+for the configuration you just specified.
+. In the popup, click **Add {agent} to your hosts** to open the **Add agent**
+flyout.
++
+TIP: If you accidentally close the popup, go to **{fleet} > Agents** and click
+**Add agent**.
+
+
+[discrete]
+[[add-agent-to-fleet-ead]]
+
+=== Install and run an {agent} on your machine
+// Include section about installing agents
+include::monitoring-install.asciidoc[tag=install-agent-widget]
+
+[discrete]
+[[view-assets-ead]]
+=== View assets
+// Include section about viewing assets
+include::monitoring-confirm.asciidoc[tag=confirm-enrollment-widget]
```

Comment:
What do you think about splitting this content into `logs` and `metrics` - Logs dashboards are common to both, and could be a shared doc.

While, sadly, both sets of assets are installed regardless of which metric collection method is used, they only have relevance in the Dashboards. 

I was wondering if we should have a an explanatory section for Dashboards, which explains what each of the dashboards are (or at least an inventory), and one for Stack Monitoring, which says that these dashboards are not populated when dashboard collection is disabled?

Reply:
Definitely worth exploring in a follow-up PR.